src/components/pages/GamePlay/GameReadStep.js

```diff
@@ -41,12 +42,15 @@ export default function GameReadStep(props) {
     <div id="read-step">
       <div id="story" className="gameplay-content">
         <div className="inner-container">
-          <div className="story-title">
-            <h2>Zoom & Boom @ A Saltwater Startup</h2>
-            <span>by Conner Burnham</span>
-          </div>
-
-          <div className="story-content">
+          <Card
```

Comment:
Nice use of the antd components. I like how you reorganized the original layout.